cmd/http/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+
+	"github.com/salt-today/salttoday2/internal/lambdas/api"
+)
+
+func main() {
+	r := chi.NewRouter()
+	r.Use(middleware.Logger)
+	r.Get("/users/{user_id}", api.GetUserCommentsHTTPHandler)
```

Comment:
can we do api/v1/users? 
Just for separation from a potential /users webpage 🤷

Reply:
Oh wait, actually there might be some miscommunication/confusion around this. 
This is what we did in salttoday1 so by far does not need to be the concrete way of doing things:
We supported getting user's comments via `/api/v1/comments?user=xyz`
Our `/api/v1/users` endpoint was for display a list of users

Reply:
I think in either case this url is a little weird, but I was going to try to make things fairly RESTY, so ideally the primary id is in the path, not the parameters